code/datums/ai_laws.dm

```diff
@@ -263,7 +263,7 @@
 			return
 
 /datum/ai_laws/proc/clear_law_sixsixsix(force)
-	if(force || !(owner && owner.mind.devilinfo))
+	if(force || !(owner && owner.has_antag_datum(/datum/antag/devil, true)))

```

Comment:
Ideally you shouldn't be directly calling has_antag_datum(); it should instead be a helper/wrapper proc like is_devil(mob) that calls has_antag_datum() and checks if the mob exists, etc